ui/js/content.js

```diff
@@ -304,6 +304,12 @@ require(['jquery', 'underscore', 'oae.core'], function($, _, oae) {
                 'setVisibility': oae.api.content.updateContent
             }
         };
+        // collabdocs have a special editor role
+        if (contentProfile.resourceSubType === 'collabdoc') {
+            widgetData.roles.editor = oae.api.i18n.translate('__MSG__CAN_EDIT__');
```

Comment:
ECMAScript standard says an Object is an `unordered collection of properties`. Looking at how we use this `roles` needs to be an array of Objects.